### PR TITLE
update apiversion of aci cni priorityclass for openshift 4.10 on esx …

### PR DESCRIPTION
**Pull request by PariyaAshok**
…flavor and above

### CODE REVIEW

**Comment by vlella** [comment]
changes LGTM
@snaiksat

File: provision/acc_provision/templates/aci-containers.yaml [hunk 1]
```diff
@@ -17,8 +17,11 @@ metadata:
 ---
 {% endif %}
 {% if config.kube_config.use_acicni_priority_class %}
-{% set list1 = config.flavor.split('-') %}
-{% if list1[1] >= "4.9"  and list1[2] == "openstack" %}
+{% set flavor_version = config.flavor.split('-') %}
+{% set major_version = flavor_version[1].split('.')[0] | int %}
+{% set minor_version = flavor_version[1].split('.')[1] | int %}
+{% if ((major_version >= 4 and  minor_version >= 9  and flavor_version[2] == "openstack") or
```
[the remark this answers is not in the document]
Comment (PariyaAshok) [reply provision/acc_provision/templates/aci-containers.yaml:23]: I think we might not have faced this issue for openshift-4.9 for ESX , thats why it is not added earlier commit.
@abhis2112 Can you please confirm this.

**Comment by snaiksat** [comment]
@PariyaAshok  thanks for the update. Why are we not seeing the flavor_openshift_49_openstack_tar being updated? Also, it seems like we are all together missing a test for OCP 4.9 on BM (I know its not related to this change, but still)?

**Comment by PariyaAshok** [comment]
> @PariyaAshok thanks for the update. Why are we not seeing the flavor_openshift_49_openstack_tar being updated? Also, it seems like we are all together missing a test for OCP 4.9 on BM (I know its not related to this change, but still)?

required change is already updated for flavor_openshift_49_openstack_tar, So its not updated.
regarding OCP 4.9 on BM, not just test we even dont have flavor i.e openshift-4.9-baremetal also.

**Review by snaiksat — APPROVED** [review]

**Comment by snaiksat** [comment]
Okay thanks @PariyaAshok , lets add the 4.9 BM separately. Also, lets backport this patch.

**Comment by PariyaAshok** [comment]
> Okay thanks @PariyaAshok , lets add the 4.9 BM separately. Also, lets backport this patch.

okay.

**Pull request merged**